utils/migration-scripts/src/sumer-giza/VideosMigration.ts

```diff
@@ -192,12 +190,10 @@ export class VideosMigration extends AssetsMigration {
       }
     )
     const feesToCover = this.assetsManager.calcDataObjectsFee(assetsParams)
+    const tx = api.tx.content.createVideo({ Member: ownerMember.id }, channelId, videoCreationParams)
     return [
       api.tx.balances.transferKeepAlive(ownerMember.controllerAccount, feesToCover),
-      api.tx.sudo.sudoAs(
-        ownerMember.controllerAccount,
-        api.tx.content.createVideo({ Member: ownerMember.id }, channelId, videoCreationParams)
-      ),
+      this.isSudo ? api.tx.sudo.sudoAs(this.sudo, tx) : tx,
```

Comment:
This will not be correctly processed by `extractFailedMigrations`, which assumes `sudoAs` is used